src/main/scala/com/kinja/amqp/AmqpConsumer.scala

```diff
@@ -206,7 +206,7 @@ class Listener[A: Reads](
 				originalSender ! ack
 			}
 		case Listener.ProcessFailed(messageBody, reason) =>
-			logger.warn(s"""[RabbitMQ] Exception while processing message "$messageBody" : $reason""")
```

Comment:
@migmit-kinja Do we want to port this change to our mainline?

Reply:
I'm not sure. Limiting the message body to 200 chars seems rather, well, limiting.

Maybe we should swap `reason` and `messageBody`? `reason` is usually short.

Reply:
Sounds reasonable.
What about the following:
`logger.warn(s"""[RabbitMQ] Exception while processing message. exception: $reason, message: $message""")`
?

Reply:
Sure.